src/components/screens/delegates/voting/voteUrlProcessor/voteUrlProcessor.js

```diff
@@ -48,7 +48,14 @@ export default class VoteUrlProcessor extends React.Component {
   componentDidUpdate() {
     if (filterObjectPropsWithValue(this.state.voteLookupStatus, 'pending').length === 0
         && this.state.pendingVotes.length > 0) {
-      this.state.pendingVotes.map(v => this.props.voteToggled(v));
+      this.state.pendingVotes.forEach(v => this.props.voteToggled({
+        account: {
+          address: v.address,
+          publicKey: v.publicKey,
+        },
+        username: v.username,
+        balance: v.balance,
```

Comment:
👍 Good catch that this also needs to be updated, but 
 💅 `balance` shouldn't be here as it is not used/defined at all in `voteToggled`:
https://github.com/LiskHQ/lisk-desktop/blob/05773664095b0304a4c851e8dd084f0b9c9a2b99/src/store/reducers/voting.js#L78-L85

Reply:
Yes, but it's already passed. I didn't want to filter a part of data out while it doesn't harm to be there.

Reply:
Remove it.

Reply:
I don't see `balance` being already passed. AFAIK it is not in delegate object at all. These are the data I see if I `console.log(this.state.pendingVotes);`
![Screenshot 2019-11-20 at 14 01 19](https://user-images.githubusercontent.com/1254342/69241094-4941aa80-0b9e-11ea-92a4-f7aa3d8552d1.png)